exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -395,6 +406,22 @@ public Viewable getProfile(@PathParam("queryid") String queryId){
     }
   }
 
+  @POST
+  @Path("/profiles/view")
```

Comment:
Good naming : `view`, not the `import`. Could you please rename all names related to the "import" to "view" ?